content/ko/docs/concepts/architecture/_index.md

```diff
@@ -57,33 +57,33 @@ description: >
 
 {{< glossary_definition term_id="kube-controller-manager" length="all" >}}
 
-컨트롤러에는 여러 가지 유형이 있다. 몇 가지 예시는 다음과 같다.
+컨트롤러에는 여러 가지 유형이 있다. 그 예시는 다음과 같다.
 
-- 노드 컨트롤러(Node Controller): 노드가 다운될 때 이를 감지하고 대응한다.
-- 잡 컨트롤러(Job Controller): 일회성 작업을 나타내는 잡(Job) 오브젝트를 감시하고, 해당 작업을 수행하기 위한 파드를 생성한다.
-- 엔드포인트슬라이스 컨트롤러(EndpointSlice controller): 엔드포인트슬라이스 오브젝트를 채워서 파드와 서비스 사이의 연결을 제공한다.
-- 서비스어카운트 컨트롤러(ServiceAccount controller): 신규 네임스페이스에 기본 서비스어카운트를 생성한다.
+- 노드 컨트롤러: 노드가 중단되었을 때 이를 감지하고 대응한다.
```

Comment:
On L62-65, I think English terms can exist since the terms appear for the first time on this doc.